docs/mddocs/Overview/install_gpu.md

```diff
@@ -44,47 +44,95 @@ We recommend using [Miniforge](https://conda-forge.org/download/) to create a py
 > [!IMPORTANT]
 > ``ipex-llm`` is tested with Python 3.9, 3.10 and 3.11. Python 3.11 is recommended for best practices.
 
-The easiest ways to install `ipex-llm` is the following commands, choosing either US or CN website for `extra-index-url`:
+The easiest ways to install `ipex-llm` is the following commands.
 
-- For **US**:
+- For **Intel Core™ Ultra Series 2 with Intel Arc™ Graphics**:
```

Comment:
add `(a.k.a. Lunar Lake)`

Reply:
Have updated :) Please have a look again. Thank you.